unstructured/ingest/interfaces.py

```diff
@@ -653,3 +653,20 @@ def session_handle(self):
     @session_handle.setter
     def session_handle(self, session_handle: BaseSessionHandle):
         self._session_handle = session_handle
+
+
+class WriteConfigSessionHandleMixin(ConfigSessionHandleMixin):
+    _session_handle: t.Optional[BaseSessionHandle] = None
+
+    @property
+    def session_handle(self):
+        """If a session handle is not assigned, creates a new one and assigns it."""
+        if self._session_handle is None:
+            self._session_handle = self.create_session_handle()
+            print("CREATE123: SESSION HANDLE")
```

Comment:
Will want to get rid of these print statements.

Reply:
Agreed, addressed with [doc, comment and logging updates](https://github.com/Unstructured-IO/unstructured/pull/1774/commits/6d84efc018bb5a2d466c06caa56705e9531d2af7)